en/core-libraries/inflector.rst

```diff
@@ -13,6 +13,57 @@ statically. Example:
 You can try out the inflections online at `inflector.cakephp.org
 <http://inflector.cakephp.org/>`_.
```

Comment:
http://inflector.cakephp.org/

> Running on CakePHP 2.9.2

This is subpar, as the 2.x API of the Inflector class does not provide dasherize() and might inflect some words slightly differently.

http://book.cakephp.org/2.0/en/core-utility-libraries/inflector.html
vs.
http://book.cakephp.org/3.0/en/core-libraries/inflector.html#Cake\Utility\Inflector::dasherize

I guess we should upgrade the inflector.cakephp.org application or provide a 3.x version side by side with the current 2.x version (preferably).